### PR TITLE
Tharun

### PR DESCRIPTION
**Pull request by tharunnayak14**

### CODE REVIEW

**Comment by Smit-create** [comment]
Why you deleted `Assigment-1.ipynb`?

**Comment by Smit-create** [comment]
I see no files added in this PR.

File: Assignment-1.ipynb [hunk 1]
```diff
@@ -0,0 +1,1131 @@
...
+    "#### Hence compute  B "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#your code here \n",
```
Comment (sin-ha) [inline Assignment-1.ipynb]: Apart from git regarding the python part this seems undone to me

**Comment by tharunnayak14** [comment]
Idk I was not able to add files

**Review by sin-ha — COMMENTED** [review]
Regarding git i guess you have downloaded the zip file instead of using git clone

If it is so

To make your repo either do  `git init` inside and follow the instructions again or clone the repo and just paste this ipynb from a text editor there.

**Comment by tharunnayak14** [comment]
Can I delete pull request and start again from scratch?
I even forked again and tried again but its not working

**Comment by sin-ha** [comment]
Yupp you could close this, try afresh and see if it works

**Pull request closed without merging**